.gitignore

```diff
@@ -10,3 +10,4 @@ npm-debug.log
 .gitconfig

```

Comment:
Can we add the `demo-credentials.js` so devs have to deliberately add it to commit changes to it?